core/config/strategyLib.js

```diff
@@ -0,0 +1,31 @@
+import { importManager } from '../../more/imports/importManager';
+
+import { natStrategy } from './strategies/natStrategy';
+import { makeUniStrategy } from './strategies/uniStrategy';
+import { makePixelStrategy } from '../../more/pixels/pixelStrategy';
+
+import { insist } from '../../util/insist';
+import { mustBeComparable } from '../../util/sameStructure';
+
+const makeCustomInsistKind = (
+  descriptionCoercer = () => true,
+) => optDescription => {
+  insist(
+    !!optDescription,
+  )`Uni optDescription must be either null or truthy: ${optDescription}`;
+  const description = descriptionCoercer(optDescription);
+  insist(!!description)`Uni description must be truthy ${description}`;
+  mustBeComparable(description);
+  return description;
+};
+
+const customInsistKind = makeCustomInsistKind();
```

Comment:
This is where we lose the description checking mentioned in the PR comment? The PR comment suggests losing this is only losing a belt-and-suspenders double check. Actually, I'm not sure whether we're even still secure without the check. TODO Need to examine.